lib/funnel/platter.ex

```diff
@@ -34,6 +34,24 @@ defmodule Funnel.Platter do
     end)
   end
 
+  @spec user_has_git_hub_repository_access?(binary, integer) :: boolean
+  def user_has_git_hub_repository_access?(user_access_token, git_hub_id) do
+    user_access_token
+    |> get_user_repository_git_hub_ids()
```

Comment:
sadly couldn't find a better strategy beyond get _all_ the users GitHub repository ids and see if the given id is in the list :(

Reply:
can we stream the ids from github instead?